otcextensions/tests/functional/osclient/nat/v2/common.py

```diff
@@ -0,0 +1,164 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+
+import json
+import uuid
+
+from datetime import datetime
+
+from openstackclient.tests.functional import base
+
+
+class NatTestCase(base.TestCase):
+    """Common functional test bits for NAT commands"""
+
+    CURR_TIME = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
+
+    def setUp(self):
+        super(NatTestCase, self).setUp()
+        UUID = uuid.uuid4().hex[:8]
+        self.ROUTER_NAME = 'sdk-test-router-' + UUID
```

Comment:
please prefix them also with some ref to NAT tests. I guess those are currently not really cleaned up, since I see orphanes

Reply:
updated naming to easily identify the orphaned resources, validated again, all allocated resources were cleaned up